src/components/auth/registration/Input.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
```

Comment:
If this component is supposed to be a reusable create a commons folder inside the components folder that's why we'll put all reusable components

Reply:
And I don't think all the props this component is receiving are required, it will be hard to reuse

Reply:
This is resolved.